### ISSUE TITLE
Diagram drew using mermaid is not exported when "save on markdown"

### ISSUE THREAD

**Issue by wli-git** [shd101wyy/markdown-preview-enhanced#1051]
no image file was exported and the exported md file looks the same with the original md file.

Thanks,
Wei

**Comment by yanshiba** [comment]
same issue , wait for solution :)

**Comment by shd101wyy** [comment]
Yeah mermaid is not currently supported when `save as markdown`. We might support it in the future. Thank you.
